scripts/upload-to-s3

```diff
@@ -33,15 +33,6 @@ main() {
     if [[ $src_hash != "$dst_hash" ]]; then
         echo "Uploading $src → $dst"
         aws s3 cp --no-progress "$src" "$dst" --metadata sha256sum="$src_hash" "$(content-type "$dst")"
-
-        if [[ $quiet == 1 ]]; then
-            echo "Quiet mode. No Slack notification sent."
-            exit 0
-        fi
-
-        if ! "$bin"/notify-slack "Updated $dst available."; then
```

Comment:
Looks like this is the only instance where `notify-slack` is being used. We can delete the script if it's no longer needed or we may replacing it with the new `send_slack_message`. Definitely not blocking this PR.

Reply:
:+1: deleted in 5d1f7ee